code/game/machinery/vending/vendor_types/crew/commanding_officer.dm

```diff
@@ -4,13 +4,16 @@ GLOBAL_LIST_INIT(cm_vending_gear_commanding_officer, list(
 		list("CAPTAIN'S PRIMARY (CHOOSE 1)", 0, null, null, null),
 		list("M46C pulse rifle", 0, /obj/effect/essentials_set/co/riflepreset, MARINE_CAN_BUY_ESSENTIALS, VENDOR_ITEM_MANDATORY),
 		list("M56C Smartgun", 0, /obj/item/storage/box/m56c_system, MARINE_CAN_BUY_ESSENTIALS, VENDOR_ITEM_MANDATORY),
+		list("XM52 tactical shotgun", 0, /obj/effect/essentials_set/co/shotgunpreset, MARINE_CAN_BUY_ESSENTIALS, VENDOR_ITEM_MANDATORY),
 
 		list("PRIMARY AMMUNITION", 0, null, null, null),
 		list("M41A MK1 Magazine", 30, /obj/item/ammo_magazine/rifle/m41aMK1, null, VENDOR_ITEM_RECOMMENDED),
 		list("M41A MK1 AP Magazine", 40, /obj/item/ammo_magazine/rifle/m41aMK1/ap, null, VENDOR_ITEM_RECOMMENDED),
 		list("M41A Extended Magazine", 20, /obj/item/ammo_magazine/rifle/extended, null, VENDOR_ITEM_REGULAR),
 		list("M41A AP Magazine", 20, /obj/item/ammo_magazine/rifle/ap, null, VENDOR_ITEM_REGULAR),
 		list("M56B Smartgun Drum", 20, /obj/item/ammo_magazine/smartgun, null, VENDOR_ITEM_REGULAR),
+		list("XM52 slug magazine", 30, /obj/item/ammo_magazine/rifle/xm52, null, VENDOR_ITEM_REGULAR),
+		list("XM52 flechette magazine", 30, /obj/item/ammo_magazine/rifle/xm52/flechette, null, VENDOR_ITEM_REGULAR),
```

Comment:
```suggestion
		list("XM52 slug magazine", 20, /obj/item/ammo_magazine/rifle/xm52, null, VENDOR_ITEM_REGULAR),
		list("XM52 flechette magazine", 20, /obj/item/ammo_magazine/rifle/xm52/flechette, null, VENDOR_ITEM_REGULAR),
```
With no way to get more ammo after expended, makin them a bit more cost effective shouldn't be a problem. Will also compensate for increasing incendiary magazine to 65 so as not to make it an easy source of incendiary shots.